Rust/Rust.sublime-syntax

```diff
@@ -633,24 +612,67 @@ contexts:
           pop: true
         - match: ','
           scope: punctuation.separator.rust
-        - include: type-any-identifier
+        - include: types-any
     - match: '\+'
       scope: keyword.operator.rust
+    - match: '\b_\b'
+      scope: keyword.operator.rust
+    - match: '!(?!\w)'
+      scope: storage.type.rust
```

Comment:
Would be great for this to get a unique subscope too which describes what this `!` type means, so that it's possible to highlight it different from builtin type names and other alphanumeric type names.